mint-auditor/src/db/models/mod.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+mod block_audit_data;
+mod block_balance;
+mod counters;
+
+pub use block_audit_data::BlockAuditData;
+pub use block_balance::BlockBalance;
+pub use counters::Counters;
```

Comment:
Optional/Equivalent: Group `pub use`
```suggestion
pub use self::{
    block_audit_data::BlockAuditData,
    block_balance::BlockBalance,
    counters::Counters,
};
```